dashboard/test/controllers/registrations_controller_test.rb

```diff
@@ -516,6 +528,306 @@ class RegistrationsControllerTest < ActionController::TestCase
     assert_equal frozen_time, sign_in.sign_in_at
   end
 
+  # USING OLD SIGN-UP FLOW [END]
+
+  # USING NEW SIGN-UP FLOW [START]
```

Comment:
> after we decide on the old/new signup flow

Is there actually a chance that we decide to keep the old flow? Asking because if we are pretty much for sure refactoring the flow, it might make sense to move the new tests over to the [spec syntax](https://www.betterspecs.org/) now while we're touching the test files. It's pretty much plug-and-play and you can add tests to an old file without changing anything in existing tests: [example](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/controllers/lti/v1/account_linking_controller_test.rb#L118-L163).